### ISSUE TITLE
Composer uses a deprecated way of authenticating against github download endpoints

### ISSUE THREAD

**Issue by stof** [composer/composer#8603]
This is the email I received from Github today:

```
Hi @stof,

On February 13th, 2020 at 09:54 (UTC) your personal access token (Composer on thorondor 2018-06-29 1220) using Composer/1.9.1 (Linux; 4.15.0-76-generic; PHP 7.3.14) was used as part of a query parameter to access an endpoint through the GitHub API:

https://api.github.com/repositories/50372644/zipball/4489d5002c49d55576fa0ba786f42dbb009be46f

Please use the Authorization HTTP header instead, as using the `access_token` query parameter is deprecated and will be removed July 1st, 2020.

Depending on your API usage, we'll be sending you this email reminder once every 3 days for each token and User-Agent used in API calls made on your behalf.
Just one URL that was accessed with a token and User-Agent combination will be listed in the email reminder, not all.

Visit https://developer.github.com/changes/2019-11-05-deprecated-passwords-and-authorizations-api/#authenticating-using-query-parameters for more information.

Thanks,
The GitHub Team
```

The announcement of the deprecation is here as a clickable link: https://developer.github.com/changes/2019-11-05-deprecated-passwords-and-authorizations-api/#authenticating-using-query-parameters%20for%20more%20information.

**Comment by glaubinix** [comment]
This has been fixed in composer 1.9.3. Unless you have a plugin installed which has it's own way of downloading files from GitHub. See for instance: https://github.com/composer/composer/issues/8586#issuecomment-584621558

**Comment by stof** [comment]
Yeah, I missed the fact that I was not on the latest composer version. So closing this issue.
